charts/aergia/values.yaml

```diff
@@ -2,7 +2,7 @@ image:
   repository: amazeeio/aergia
   pullPolicy: IfNotPresent
   # Overrides the image tag whose default is the chart appVersion.
-  tag: ""
+  tag: k8s_122
```

Comment:
do we have an updated image for 1.22 - the appVersion says v0.1.2? - we should remove this tag if so?

Reply:
Probably remove it